### PR TITLE
Fix ARM compatibility #22

### PR DESCRIPTION
**Pull request by Derpzz**
This PR addresses issue #22 by updating all Docker images to the latest versions that support ARM. While I was able to run this successfully on my ARM M1 MacBook, I do not have a reference of the prior state. Therefore, please test extensively before merging.

### CODE REVIEW

File: docker/images/java-agent/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:17-alpine
+FROM eclipse-temurin:22-alpine
```
Comment (uhafner) [inline docker/images/java-agent/Dockerfile]: This Java agent is supposed to run Java 17, does it not work on your architecture?

File: docker/images/java-11-agent/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:11-alpine
+FROM eclipse-temurin:22-alpine
```
Comment (uhafner) [inline docker/images/java-11-agent/Dockerfile]: This Java agent is supposed to run Java 11, does it not work on your architecture?

File: docker/images/java-agent/Dockerfile (hunk 1 above)
Comment (uhafner) [reply docker/images/java-agent/Dockerfile]: I think I need to add another agent that runs Java 21

**Review by uhafner — CHANGES_REQUESTED** [review]
I think it makes sense to separate the controller changes from the agent changes...

**Comment by Derpzz** [comment]
I reverted all agent changes. Now this PR only contains the controller version bump. All agent changes will be in a separate PR.

**Review by uhafner — APPROVED** [review]

**Pull request merged**